openreview/venue_request/process/request_form_pre_process.py

```diff
@@ -15,4 +15,20 @@ def process(client, note, invitation):
         raise openreview.OpenReviewException('Assigned area chairs must see the reviewer identity')
 
     if 'Yes, our venue has Senior Area Chairs' in note.content.get('senior_area_chairs', '') and 'All Senior Area Chairs' not in note.content['reviewer_identity'] and 'Assigned Senior Area Chair' not in note.content['reviewer_identity']:
-        raise openreview.OpenReviewException('Assigned senior area chairs must see the reviewer identity')
\ No newline at end of file
+        raise openreview.OpenReviewException('Assigned senior area chairs must see the reviewer identity')
+
+    withdrawal_expiration = note.content.get('withdraw_submission_expiration')
+    if withdrawal_expiration:
+        try:
+            withdrawal_expiration = datetime.datetime.strptime(withdrawal_expiration, '%Y/%m/%d %H:%M')
+        except ValueError:
+            withdrawal_expiration = datetime.datetime.strptime(withdrawal_expiration, '%Y/%m/%d')
+
+        submission_deadline = note.content.get('Submission Deadline')
+        try:
+            submission_deadline = datetime.datetime.strptime(submission_deadline, '%Y/%m/%d %H:%M')
+        except ValueError:
+            submission_deadline = datetime.datetime.strptime(submission_deadline, '%Y/%m/%d')
+
+        if withdrawal_expiration < (submission_deadline + datetime.timedelta(minutes=30)):
```

Comment:
why is the withdrawal expiration date at the same time than the submission deadline? is it because they don't want authors to withdraw their submissions?

Reply:
I'm thinking that instead of validating this, we should adjust the withdrawal dates.

If withdrawal_expiration < submission_deadline + 30 min then don't event create the withdrawal invitation or set the cdate to be the same than the expiration